### DISCUSSION TITLE
Delta Pro missing standby switch status entity?

### DISCUSSION THREAD

**Discussion by skpow55** [vwt12eh8/hassio-ecoflow#37] [Q&A]
I just noticed that I don't see a standby mode status entity for my Delta Pro with the HA integration.  Anyone else notice this?

**Comment by Ne0-Hack3r** [comment]
Confirmed. Is that value presented in the data stream? It's obviously a manual switch on the unit just like the custom charging switch so likely can't be "controlled" from HA... but perhaps that status can be exposed as an entity?

**Comment by vwt12eh8** [comment]
I don't think I was notified of this condition when I first investigated. 
I'll check again next time I have some free time.
